page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,160 @@
+<script>{
+  "title": "Using jQuery",
+  "level": "beginner"
+}</script>
+
+### jQuery Fundamentals
+
+This tutorial will help you try out some common uses of jQuery.
+
+If you haven't setup a demo HTML page that includes a copy of jQuery, see [How To Setup jQuery](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/) for the complete tutorial on how to get started, or simply follow along with the examples below.
+
+### Adding and Removing an HTML Class
+
+jQuery is often used to bind events to elements in the DOM, assigning them functionality to be triggered by a user's actions on the page. To demonstrate, let's highlight the menu item a user last clicked by changing its HTML class to "active", and assigning the "active" class a different appearance using CSS.
+
+First, copy or create the `index.html` file below:
+
+```
+// index.html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><a href="http://jquery.com">Home</a></li>
+      <li><a href="http://google.com">About</a></li>
+      <li><a href="http://github.com">Products</a></li>
+    </ul>
+  </nav>
+  <script src="jquery.js"></script>
+  <script src="demo.js"></script>
+  <script src="main.css"></script>
```

Comment:
My memory isn't good but I don't think a CSS file is included with a `script` element. :stuck_out_tongue_winking_eye: